reference/api/search.md

```diff
@@ -735,3 +738,21 @@ Queries using `_geoPoint` will always include a `geoDistance` field containing t
 ```
 
 [You can read more about location-based sorting in our dedicated guide.](/learn/advanced/geosearch.md#sorting-results-with-geopoint)
+
+### Matching strategy
+
+**Parameter**: `matchingStrategy`
+**Expected value**: `last` or `all`
+**Default value**: `last`
+
+Defines the strategy used to match query terms in documents. `last` returns documents containing all the query words first. If there aren't enough documents to meet the requested `limit`, it will remove one query term at a time, starting from the end of the query.
+
+<CodeSamples id="search_parameter_guide_matching_strategy_1" />
+
+With the above code sample, Meilisearch will first return documents that contain all three words. If the results don't meet the requested `limit`, it will return documents with just `big fat` followed by just `big`.
+
+`all`, on the other hand, only returns documents that contain all query terms. Meilisearch will not match any more documents even if there aren't enough to meet the requested `limit`. Searching for `big fat liar` will only return documents that contain all three words.
```

Comment:
We want to avoid ambiguous constructions like "on the other hand" in our API reference. We don't know what the reader may have read before this: he or she could be speed-reading, or could have followed a link directly to this paragraph. Either way they would be missing the necessary context to understand what we mean by "on the other hand". (Not to mention that we may not always have only two values for this parameter)

```suggestion
#### `all`

`all` only returns documents that contain all query terms. Meilisearch will not match any more documents even if there aren't enough to meet the requested `limit`. 
```

Reply:
Thanks for the tips Tommy!